doc/examples/test_serialization.cpp

```diff
@@ -112,6 +136,8 @@ auto main() -> int {
 
   Thread::cpu = nullptr;
   Thread::apu = nullptr;
-  munmap((void*)0x900000000, 2 * 65536);
+  #ifndef _WIN32
+    munmap((void*)0x900000000, 2 * 65536);
+  #endif
```

Comment:
I think we should call `UnmapViewOfFile` on Windows, like we call `munmap` on POSIX. Sure, the process is about to exit which means the data will be automatically unmapped anyway, but it's good for example code to be extra careful.